.htaccess

```diff
@@ -3,6 +3,8 @@ RewriteEngine On
 # support for old URLs
 RewriteRule ^(.*)/security-slider.html$ /$1/security-settings/ [R=302,L]
 RewriteRule ^security-slider.html$ /security-settings/ [R=302,L]
+# strip '.html' and redirect
+RewriteRule ^(.*)/(.*).html$ /$1/$2/ [r=302,L]
```

Comment:
The current version makes users that visit https://tb-manual.torproject.org/security-slider.html to end up in https://tb-manual.torproject.org/security-settings/, but your pull request will make them go to https://tb-manual.torproject.org/security-slider/
Please make it that they still go to https://tb-manual.torproject.org/security-settings/